includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,292 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor HTML widget.
+ *
+ * Elementor widget that insert a custom HTML code into the page.
+ *
+ * @since 1.0.0
+ */
+class Widget_Playing_Cards extends Widget_Base {
+
+	/**
+	 * Get widget name.
+	 *
+	 * Retrieve HTML widget name.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget name.
+	 */
+	public function get_name() {
+		return 'playing-cards';
+	}
+
+	/**
+	 * Get widget title.
+	 *
+	 * Retrieve HTML widget title.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget title.
+	 */
+	public function get_title() {
+		return esc_html__( 'Playing Cards', 'elementor' );
+	}
+
+	/**
+	 * Get widget icon.
+	 *
+	 * Retrieve HTML widget icon.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 * @return string Widget icon.
+	 */
+	public function get_icon() {
+		return 'eicon-code';
+	}
+
+	/**
+	 * Get widget keywords.
+	 *
+	 * Retrieve the list of keywords the widget belongs to.
+	 *
+	 * @since 2.1.0
+	 * @access public
+	 *
+	 * @return array Widget keywords.
+	 */
+	public function get_keywords() {
+		return [ 'playing cards', 'card' ];
+	}
+
+	/**
+	 * Register HTML widget controls.
+	 *
+	 * Adds different input fields to allow the user to change and customize the widget settings.
+	 *
+	 * @since 3.1.0
+	 * @access protected
+	 */
+	protected function register_controls() {
+		$card_numbers = [
+			'A' => ['title' => esc_html__( 'A', 'elementor' )],
```

Comment:
You'll need spaces after square brackets for our code style. 
```suggestion
			'A' => [ 'title' => esc_html__( 'A', 'elementor' ) ],
```